### PR TITLE
Update introduction.rst

### PR DESCRIPTION
**Pull request by nchampollion**
1. Do you think that a preprocessing section into Principles could be useful ? Explaining RGI, projection and DEM ...
2. In the Climate data and mass-balance section, is the word "interpolated" correct ? Because I guess we chose the gridded climate data closest to he glacier location.

Closes https://github.com/OGGM/oggm/issues/###

- [ ] Tests added/passed
- [ ] Fully documented
- [ ] Entry in `whats-new.rst`

### CODE REVIEW

File: docs/introduction.rst [hunk 1]
```diff
@@ -4,8 +4,11 @@ Introduction
 
 We illustrate with an example how the OGGM workflow is applied to the
 Tasman Glacier in New Zealand (see figure below).
-Here we describe shortly the purpose of each processing step, while more
-details are provided in the other sections:
+Here we describe shortly the purpose of each processing steps, while more
```
Comment (fmaussion) [inline docs/introduction.rst:7]:
```suggestion
Here we describe shortly the purpose of each processing step, while more
```

I think singular is correct here?

File: docs/introduction.rst [hunk 2]
```diff
@@ -4,8 +4,11 @@ Introduction
...
+details are provided in the other sections and the theory of glacier physic
+can for example be found in `Cuffey and Paterson, (2010)`_.:
+
+.. _Cuffey and Paterson, (2010): https://www.elsevier.com/books/the-physics-of-glaciers/cuffey/978-0-12-369461-4
```
Comment (fmaussion) [inline docs/introduction.rst:11]: Maybe not scare people with Cuffey and Paterson right in the general intro? I like the reference to the other sections though.

File: docs/introduction.rst [hunk 3]
```diff
@@ -37,10 +40,10 @@ Climate data and mass-balance
 
 Ice thickness inversion
   Using the mass-balance data computed above and relying on mass-conservation
-  considerations, an estimate of the ice flux along each glacier cross-section
-  can be computed, by making assumptions about the shape of the cross-section
-  (parabolic or rectangular) and using the physics of ice flow, the model
-  computes the thickness of the glacier along the flowlines and the total
+  considerations, an estimate of the ice flux along each glacier grid point cross-section
+  is computed by making assumptions about the shape of the cross-section
+  (parabolic or rectangular). Using the physics of ice flow and the shallow ice approximation,
```
Comment (fmaussion) [inline docs/introduction.rst:45]:
```suggestion
  (parabolic, rectangular or trapezoid). Using the physics of ice flow and the shallow ice approximation, 
```

**Review by fmaussion — COMMENTED** [review]
Great, thanks! I'm open for other suggestions! I found it quite hard to describe everything, but if you have an idea please go for it!!!!


Interpolated for climate is not strictly valid no, although "nearest neighbor" IS interpolation, albeit a very simple one.

File: docs/introduction.rst (hunk 1 above)
Comment (nchampollion) [reply docs/introduction.rst:7]: I think both are correct. However, it makes probably more sense to link the singular for step with the singular of purpose ...

**Comment by fmaussion** [comment]
Merging now - thanks! I think some of the suggestions you made here or in the other PR are good, feel free to implement them if you think its important!

File: docs/introduction.rst (hunk 2 above)
Comment (nchampollion) [reply docs/introduction.rst:11]: I understand not to scare people. However it makes more strength to the model with this reference ?

**Comment by nchampollion** [comment]
Ok, great I'll see if I put later on more information in the doc :-)

**Pull request merged**